### DISCUSSION TITLE
How to highlight specific element?

### DISCUSSION THREAD

**Discussion by natanfelles** [PrismJS/prism#3590] [Q&A] [answered]
Hello!

I would like to know if it is possible to highlight code blocks within a single element and not on the whole page?

I'm working on a debug bar (which loads PrismJS) and would like to avoid conflicts with Prism or Highlightjs used on the page.

Thank you very much!

**Comment by RunDevelopment** [comment, marked as answer]
Yes, use [`Prism.highlightElement`](https://prismjs.com/docs/Prism.html#.highlightElement).

**Reply by natanfelles** [reply]
Sorry. I hadn't seen a link to the documentation. `Prism.highlightAllUnder` solved my problem. Thank you very much!
